src/components/Header/Header.tsx

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { NewTodo } from '../NewTodo';
+
+interface Props {
+  todos: Todo[];
+  toggleStatus: boolean;
+  onUpdateAllTodosComplete: () => Promise<void>
+}
+
+export const Header: React.FC<Props> = React.memo(({
+  todos,
+  toggleStatus,
+  onUpdateAllTodosComplete,
+}) => {
+  return (
+    <header className="todoapp__header">
+      {todos.length > 0 && (
+        <button
+          type="button"
```

Comment:
Add an aria-label attribute to the button to improve accessibility and comply with jsx-a11y/control-has-associated-label rule.

Reply:
thx a lot! greate idea! I didn't understand how to fix it and just turn off linter((